account/client.go

```diff
@@ -313,7 +315,9 @@ func (this *ClientImpl) getAccount(accData *AccountData, passwd []byte) (*Accoun
 	if !accData.VerifyPassword(passwd) {
 		return nil, fmt.Errorf("verify password failed")
 	}
+	keypair.SetScryptParam(this.walletData.Scrypt)
 	privateKey, err := keypair.DecryptPrivateKey(&accData.ProtectedKey, passwd)
+	keypair.SetScryptParam(nil)
```

Comment:
changed to default security here?

Reply:
Every encryption/decryption operation set the parameters as needed, and reset to default afterwards.